agent/agent-tooling/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorTraceExporter.java

```diff
@@ -227,9 +219,12 @@ private void exportRemoteDependency(
     RemoteDependencyTelemetryBuilder telemetryBuilder = RemoteDependencyTelemetryBuilder.create();
     initTelemetry(telemetryBuilder);
 
+    float samplingPercentage = 100; // TODO
```

Comment:
the baseline datamodel handle this for all telemetry. not sure why you need to do this specifically?

Reply:
it would be better to check config's sampling setting first.

Reply:
i also noticed TelemetryUtil.getSamplingPercentage.

Reply:
ya, we don't support samplingPercentage in this exporter yet (that's what the TODO is for), I'm just sync'ing the code between the two span exporters as much as possible, then will try to unify the codebase